### DISCUSSION TITLE
Dimension Stack on Servers

### DISCUSSION THREAD

**Discussion by TTPain** [iPortalTeam/ImmersivePortalsMod#1363] [Q&A] [answered]
Is it possible, or how would I, start a new world with dimension stack enabled? I would like to Have a server that has multiple dimensions stacked from the get go and not have to attempt to transfer a world to it. Is there some config or comment I can add to the server.properties that will allow it to Stack during generation?

**Comment by qouteall** [comment, marked as answer]
https://qouteall.fun/immptl/wiki/Dimension-Stack.html#how-to-use-dimension-stack-on-a-server

**Reply by darthzapp** [reply]
It also crashes when applying a dimensional stack in single-player with: Internal Exeption: io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:custom_payload'

**Comment by Gameking1happy** [comment]
this does not work, server kicks for packet being bigger than expected

**Reply by darthzapp** [reply]
I have the same problem

**Reply by Gameking1happy** [reply]
@darthzapp best workaround is to just download the world from the server, add it to client, and do the command there, then upload the changed world up. Note that I had an issue where player data was as client player (being in the level.dat, not player data folder), but this world I had originally on client, and moved to server, so possibly could be fine otherwise, but if you have in level.dat player data, delete it, as it will load that and possibly overwrite your existing player data, and do so after you finish the changes just in case you gotta do that again.
